assembl/static2/js/app/components/administration/discussion/timelineConfiguration/index.jsx

```diff
@@ -129,8 +129,9 @@ class TimelineFields extends React.Component<Props> {
                               picker={{ pickerType: I18n.t('administration.landingPage.header.startDate') }}
                               editLocale={editLocale}
                               placeHolder={startDatePickerPlaceholder}
-                              showTime={false}
-                              dateFormat="LL"
+                              showTime
```

Comment:
`showTimeSelect` no?

Reply:
in the datePickerAdapter it's showTimeSelect={showtime}